.github/workflows/check_dependencies.yaml

```diff
@@ -3,6 +3,7 @@ name: Check dependencies
 on:
   schedule:
     - cron: "0 9 * * 1"
+  workflow_dispatch:
```

Comment:
This allows us to trigger the workflow manually, correct?

Reply:
Yes, it's useful for testing and doesn't really hurt, so I left it in.